### PR TITLE
Added sort linked list of zeros, ones and twos in cpp

### PR DESCRIPTION
**Pull request by shriya030**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added a problem sort linked list of zeros, ones and twos in cpp.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5381

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Why the code matches with this [code ](https://github.com/abhinavjdwij/GeeksforGeeks-Solutions/blob/master/Given%20a%20linked%20list%20of%200s%2C%201s%20and%202s%2C%20sort%20it.cpp)

**Comment by shriya030** [comment]
> Why the code matches with this [code ](https://github.com/abhinavjdwij/GeeksforGeeks-Solutions/blob/master/Given%20a%20linked%20list%20of%200s%2C%201s%20and%202s%2C%20sort%20it.cpp)

I had tried this question earlier by myself. I was not at all aware that this code is available on the internet. This approach is similar to that of sorting an array of 0's, 1's, and 2's which uses counting of 0's 1's, and 2's and then changing the array.

File: C-Plus-Plus/ds/sort_list_of_0s_1s_2s.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+/*
+  Input:
+  n - number of elements in the list
+  n elements
+  Example:
+  0 1 2 1 2
+
+  Output:
+  0 1 1 2 2
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/ds/sort_list_of_0s_1s_2s.cpp]: remove this and,

Use proper display message how you are reading and writing outputs

Add input-output prompts to make the program more interactive.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
also add one line short description of each function

File: C-Plus-Plus/ds/sort_list_of_0s_1s_2s.cpp (hunk 1 above)
Comment (shriya030) [reply C-Plus-Plus/ds/sort_list_of_0s_1s_2s.cpp]: @ankitaggarwal23 made the required changes
Please check.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by shriya030** [comment]
> Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

@ankitaggarwal23 Is it okay?

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**